zilliqa/src/api/types/zil.rs

```diff
@@ -619,6 +620,42 @@ pub struct TransactionReceiptResponse {
     pub success: bool,
 }
 
+// From https://github.com/Zilliqa/Zilliqa/blob/master/src/common/TxnStatus.h#L23
+#[derive(Serialize_repr, Deserialize_repr, Clone)]
+#[repr(u8)] // Because otherwise it's weird that 255 is a special case
+pub enum TxnStatusEnum {
```

Comment:
Nittiest of nit picks - We know its an `enum` already so just call it `TxnStatus` :)

Reply:
I thought about that, but I wanted to make it clear that it's not the whole transaction status - the return type of get_transaction_status isn't just that enum, it's the whole of TransactionStatusResponse, with various other fields too. I agree that enum isn't the nicest way of putting it though, so I'm just about to push a commit changing it to TxnStatusCode, if that works for you?